--- cln-1.1.8/src/Makefile.in.orig	2004-05-01 00:14:41.000000000 +0200
+++ cln-1.1.8/src/Makefile.in	2004-08-03 00:11:45.916984829 +0200
@@ -202,7 +202,7 @@
 $(ALLOBJECTS_LO) : $(SUBDIRS_TARGET)
 
 libcln.la : $(LIBTOOL) $(ALLOBJECTS_LO)
-	$(LIBTOOL_LINK) $(CC) -o libcln.la -rpath $(libdir) -version-info $(CL_CURRENT):$(CL_REVISION):$(CL_AGE) $(LDFLAGS) $(LIBS) $(ALLOBJECTS_LO)
+	$(LIBTOOL_LINK) $(CXX) -o libcln.la -rpath $(libdir) -version-info $(CL_CURRENT):$(CL_REVISION):$(CL_AGE) $(LDFLAGS) $(LIBS) $(ALLOBJECTS_LO)
 
 install : all force
 	$(MKDIR) $(DESTDIR)$(libdir)
